example/lib/main.dart

```diff
@@ -397,10 +570,6 @@ class _DartCodeViewerPage extends StatelessWidget {
     );
   }
 }
-
-      ''',
-       ),
-      ),
-    );
-  }
-}
+  
+''';
+}
```

Comment:
Nit: add a new line at the end of the file.

Reply:
Done